lib/predictive_load/config.rb

```diff
@@ -0,0 +1,38 @@
+module PredictiveLoad
+  module Config
+    class ConfigError < ::StandardError
```

Comment:
FYI could use `ArgumentError` to simplify

Reply:
Good point, thanks 👍

Reply:
Done.